lib/stat_tracker.rb

```diff
@@ -1,6 +1,8 @@
 require 'csv'
 require './creator'
+require_relative './modules/league_stats'
 class StatTracker
+  include LeagueStats
```

Comment:
Nice job adding the module. We can now include our methods into the StatTracker class!

Reply:
Yes this will be perfect in keeping a clean code!